src/main/webapp/app/config/constants/constants.ts

```diff
@@ -427,3 +432,10 @@ export const DEFAULT_TOAST_ERROR_OPTIONS: ToastOptions = {
 export const KEYCLOAK_LOGOUT_REDIRECT_PARAM = 'post_logout_redirect_uri';
 export const KEYCLOAK_SESSION_TERMINATED_PARAM = 'session_terminated';
 export const KEYCLOAK_UNAUTHORIZED_PARAM = 'unauthorized';
+
+/**
+ * LocalStorage Keys
+ * Use getLocalStorageKey util method to attach application name to key
+ */
+export const PRIORITY_ENTITY_MENU_ITEM_KEY = 'entityMenuPriorityKey';
+export const SOMATIC_GERMLINE_SETTING_KEY = 'somaticGermlineSettingKey';
```

Comment:
@zhx828 I think the naming for localStorage keys should be `${appName}-{key}` to reduce the possibility of name clashes across different applications.

Also, do you prefer a util method or defining all localstorage key in constants with a comment with how they should be named? I find it easy to accidently not use the `getLocalStorageKey` util.

Reply:
@calvinlu3 agree with the suggestion on the key format.
Personally I think we should have a local storage store which it will manage all the keys. Fine for me to use constants for now.